variables.tf

```diff
@@ -16,12 +16,6 @@ variable "resource_group_name" {
   nullable    = false
 }
 
-variable "use_for_each" {
-  type        = bool
-  description = "Use `for_each` instead of `count` to create multiple resource instances."
-  nullable    = false
-}
-
```

Comment:
I know its not a part of your PR but would you mind removing the default value of the `address_space` in line 27. We would like to enforce users to pass in value to this rather than defaulting